package.json

```diff
@@ -43,6 +43,7 @@
   },
   "homepage": "https://github.com/Xiphe/cachified#readme",
   "devDependencies": {
+    "@cloudflare/workers-types": "^4.20231025.0",
```

Comment:
I think if we use the package within adapters.ts it should be installed as a dependency. (Unless the build process will bundle the types [please double-check]). 

That said, I'm proud that this package does not bring any dependencies when installing and I would like to keep it this way. 
For the other adapters we've modeled the types of the external API on our end (see [`Redis3LikeCache`](https://github.com/Xiphe/cachified/blob/main/src/adapters.ts#L35-L48)). Maybe that's an option here, too?

Reply:
As it stands, this package is only used to provide type information, the import specifies everything imported as type only
```ts
import type { KVNamespace, ExecutionContext } from '@cloudflare/workers-types';
```

In the final build output, the `dist/index.cjs` and `dist/index.mjs` files do not contain an import from this package.

That being said, the type output generated does use it, so `dist/src/adapters.d.ts` uses the package to provide type information.

Given this use case, maybe this is a better approach
```js
{
    "peerDependencies": {
      "@cloudflare/workers-types": "^4.20231025.0"
    },
    "peerDependenciesMeta": {
      "@cloudflare/workers-types": {
        "optional": true
      }
    },
    "devDependencies": {
      "@cloudflare/workers-types": "^4.20231025.0",
    }
  }
```

This way we clearly mark that you need to install this package yourself if you want the additional type information, but given that it just provides typing data it's an optional install.

Reply:
Yeah, this seems feasible. I'd still prefer to maintain a minimal interface of the API we expect on our ends.